src/includes/class_reservationPermissions.php

```diff
@@ -321,7 +368,9 @@ public function insertRecord($id, $type, $room, $email){
             throw new Exception("Invalid ID provided.");
         }
 
-        if (self::duplicatePermissionsCheck($id, $type, $room, $email) === FALSE) {
+        $dupCheck = self::duplicatePermissionsCheck($id, $type, $room, $email);
+
+        if ($dupCheck === FALSE) {
```

Comment:
This change should be reverted. The variable assignment increases memory usage, as well as the time required for the assignment. There isn't any reason for breaking out the variable. 

PHP allows assignment in the if statement as well, if needed. To make the code more concise. So, if you did need the variable you could do:
if (($dupeCheck = self::duplicatePermissionsCheck($id, $type, $room, $email)) === FALSE) {
}

Also, eliminating the "e" just makes it harder to read, with no benefit. it is a single letter. dupeCheck is more readable and doesn't take any longer to type.